mangdangroboticsclub/minipupper_ros
# Update for README & .repos & other documentations

## Proposed changes
I have adjusted the .repo configuration and deleted documentation and license related to Champ.
I have also reformed README.md for the new navigation and bringup user guide.
### Types of changes

- [X] Documentation update and .repo configuration adjustment.

## Testing and Verification
To verify these changes, I performed the following tests:

1. check the new README.
2. check the .repo file.
3. check other documentation's modifications.

## Test Configuration

__Mini Pupper Version__  
Mini Pupper

__Raspberry Pi OS + ROS version__  
Ubuntu 22.04, ROS 2 Humble

__PC OS + ROS version__  
Ubuntu 22.04, ROS 2 Humble

## Checklist
